src/templates/chat/license.html

```diff
@@ -1,23 +1,21 @@
 <div class="ose game-license">
-  <p class="ose game-license">
-    Unofficial Old-School Essentials on Foundry VTT <!-- BEGIN TEXT REQUIRED BY LICENSE -->requires <strong><em>Old-School Essentials</em></strong>.<!-- END TEXT REQUIRED BY LICENSE --> Find the game at <a href="https://necroticgnome.com">Necrotic Gnome's website</a>.
+  <p>
+    Simple, robust rules compatible with decades of B/X and OSR content. <a href="https://necroticgnome.com">See the website for full information about Old-School Essentials.</a>
```

Comment:
I have opinions about what text to surround with links, but not enough to block the PR. Thoughts on making just the call to action ("See the website") the link?

Reply:
I lifted that sentence, including the link, exactly from the NG website. But it was neither an expectation from NG nor a goal to be exact, it was just easier to use "vetted" language

In the interest of brevity we can cut it down